packages/yarnpkg-builder/sources/tools/makeConfig.ts

```diff
@@ -30,17 +29,14 @@ export const makeConfig = (config: webpack.Configuration) => merge({
       exclude: /\.d\.ts$/,
       use: [{
         loader: require.resolve(`babel-loader`),
```

Comment:
Currently doesn't do anything, but we can keep it to allow users to provide their own babel config